src/convertToObject.js

```diff
@@ -6,7 +6,24 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssStyle = {};
+  const arrStr = sourceString.split(';');
+
+  for (const pieceArr of arrStr) {
+    // @ts-ignore
+    // eslint-disable-next-line no-console
+    console.log(pieceArr);
+
+    if (pieceArr) {
+      const [styleName, value] = pieceArr.trim().split(':');
```

Comment:
Ensure that the `pieceArr.trim().split(':')` operation correctly handles cases where there might be extra spaces or missing colons. Consider adding validation or error handling for malformed entries.